ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -6800,6 +6795,30 @@ let pool_get_guest_secureboot_readiness printer rpc session_id params =
        (Record_util.pool_guest_secureboot_readiness_to_string result)
     )
 
+let pool_sync_bundle fd _printer rpc session_id params =
+  let filename = List.assoc "filename" params in
```

Comment:
I would prefer `List.assoc_opt` and a dedicated internal failure in case none is found.